trainingMaquetado/Maquetado.css

```diff
@@ -0,0 +1,49 @@
+hr {
+	background-color: #F5C85F;
+	width: 4px;
+	height: 300px;
+	margin: 60px;
+	border: none;
+  border-radius: 60px;
+  flex-shrink: 0; 
+  background: radial-gradient(#F5C85F 40%, white 60%);
```

Comment:
I would've used a linear gradient that starts with white, continues with orange and finishes with white,
```
 background: linear-gradient(white, #F5C85F, white);
```

but what you achieved looks good too